src/Messages/Traits/ArrayAccessTrait.php

```diff
@@ -20,6 +21,7 @@ public function offsetExists($offset)
     /**
      * {@inheritdoc}
      */
+    #[\ReturnTypeWillChange]
```

Comment:
Why do we need to repeat this all over?

Reply:
Because the methods provided in this trait are used in our classes that implement `\ArrayAccess` and they're missing the return type. 
Since we're moving to ePoetry and this component will most likely become obsolete there is no need to fix all the methods.